package.json

```diff
@@ -1,82 +1,108 @@
 {
```

Comment:
It looks like your editor might have re-indented this file. It's a bit hard to review the changes here with all the whitespace changes. Could you revert the indentation changes?